playground/web/index.html

```diff
@@ -0,0 +1,382 @@
+                        </div>
+                        <div class="flag-group" style="display: none;">
+                            <input type="checkbox" id="flag-unicode" checked>
+                            <label for="flag-unicode">Unicode</label>
+                        </div>
+                    </div>
+                </div>
+
+                <div class="input-group">
+                    <label for="debounce-delay">Update Delay (ms):</label>
+                    <input type="number" id="debounce-delay" value="300" min="100" max="2000" step="100" style="height: auto;">
+                </div>
+
+                <div class="input-group">
+                    <label for="text-input">Test Text:</label>
+                    <textarea id="text-input" placeholder="Enter the text you want to test the regex against..."></textarea>
+                </div>
+
+                <div class="toggle-section">
+                    <button id="show-parse-tree" class="toggle-btn">Show Parse Tree</button>
+                    <button id="show-analysis" class="toggle-btn">Show Analysis</button>
+                </div>
+            </div>
+        </div>
+
+        <!-- Output Panel -->
+        <div class="panel">
+            <div class="panel-header">Results</div>
+            <div class="panel-content">
+                <div class="output-container">
+                    <div class="output-section">
+                        <div class="output-header">Match Results</div>
+                        <div id="match-results" class="output-content">
+                            <div class="info">Enter a regex pattern and test text to see results</div>
+                        </div>
+                    </div>
+
+                    <div class="output-section">
+                        <div class="output-header">Highlighted Text</div>
+                        <div id="highlighted-text" class="output-content">
+                            <div class="info">Matches will be highlighted here</div>
+                        </div>
+                    </div>
+
+                    <div id="parse-tree-section" class="output-section hidden">
+                        <div class="output-header">Parse Tree</div>
+                        <div id="parse-tree-content" class="output-content">
+                            <div class="parse-tree" id="parse-tree-display"></div>
+                        </div>
+                    </div>
+
+                    <div id="analysis-section" class="output-section hidden">
+                        <div class="output-header">Analysis</div>
+                        <div id="analysis-content" class="output-content">
+                            <div class="parse-tree" id="analysis-display"></div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+
+    <footer>
+        <span id="version">v0.16.1</span> | 
```

Comment:
Should this just be a placeholder and the GitHub actions build should replace it with the actual version on build?

Reply:
yep, fixed